src/alphafold_impact/pipelines/data_collection_oa/utils.py

```diff
@@ -262,6 +262,22 @@ def _create_concept_year_filter(concept_ids: List[str], years: List[int]) -> str
     return ",".join(filter(None, [year_filter, concept_filter]))
 
 
+def chunk_list(input_list: List[str], chunk_size: int) -> List[List[str]]:
```

Comment:
This may be worth addressing now and not in the future - I can do it if you want. 

The functionality of this is equivalent to `_chunk_work_ids`, as you first split the chunks, and then append them in `_create_concept_year_filter`. Note that the filter condition (`publication_year` or `publication_year`) I control for in the the function `_works_generator` (itself part of another internal function and one of three functions (lazy, eager, parallel). The argument is `filter_criteria`. 

Against, nothing wrong, but for legacy purposes or future work on this, it would be good to harmonise. I can do it, if you're busy. Just let me know :)